index.html

```diff
@@ -16,19 +17,22 @@
     </script>
     <title>Hello, I am Adam Chang</title>
 </head>
-<body>
+<body>    
     <header class="head-container">
          
         <a href=""><h2 id="name">Adam Chang</h2></a>
-        <nav>
+        <nav id="nav">
             <ul class="nav-list">
-                <li class="nav-list-items"><a href="">Portfolio</a></li>
-                <li class="nav-list-items"><a href="">About</a></li>
-                <li class="nav-list-items"><a href="">Contact</a></li>
-                <li class="nav-list-items"><a href=""><img src="Images/mail.svg" alt=""></a></li>
+                <li class="nav-list-items"><a class="nav_a" href="#works-container">Portfolio</a></li>
+                <li class="nav-list-items"><a class="nav_a" href="#about_me">About</a></li>
+                <li class="nav-list-items"><a class="nav_a" href="#contact-form">Contact</a></li>
+                <li class="nav-list-items"><a class="nav_a" href=""><img src="Images/mail.svg" alt=""></a></li>
 
             </ul>            
-            <img  id="menu-btn" src="Images/Union.png" alt="menu button"> 
+            <button id="menu-btn" type="button">
+                <img class="menu-icon"  src="Images/Union.png" alt="menu button">
+                <img class="close-icon" src="Images/close.png" alt="close button">
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close buttons on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.

Reply:
Thank you for the suggestion, I will definitely do that. Thank you for the review!